app/views/users/show.html.erb

```diff
@@ -1,13 +1,20 @@
-<h1>Post Details</h1>
-<% @users.each do |user| %>
-    <h1>List of Posts for User: <%= user.name %></h1>
-    <ul>
-      <% user.posts.each do |post| %>
-        <li><%= post.title %></li>
-        <li><%= post.comments %></li>
-      <% end %>
-    </ul>
-  <% end %>
-<p>Title: Example Post</p>
-<p>Author: John Doe</p>
-<p>Content: Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
\ No newline at end of file
+<h1>Users Info</h1>
+<h1><%= @users.name %></h1>
+<div><img src="<%= @users.photo %>" alt="User Photo"></div>
+Number of posts: <%= @users.posts_counter %><br>
+<h2>Bio</h2>
+<%= @users.bio %>
+
+<% @posts.limit(3).each do |post| %>
+  <li><%= post.title %></li>
+  <li><%= post.text %></li>
+  <li>Comments: <%= post.comments_counter %></li>
+  <li>Likes: <%= post.likes_counter %></li>
+<% end %>
+
```

Comment:
[OPTIONAL] It would be best if you made the post title clickable so that it can redirect a user to the post-show page.